app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,12 @@
+.alert-alert {
+  @extend .alert-warning;
+}
+
+.alert-notice {
+  @extend .alert-info;
+}
+
+.alert {
+	text-align: center;
+	margin-bottom: 0;
+}
```

Comment:
Please configure your editor to insert newlines at the end of the file automatically. If you're using Sublime, the configuration option is `ensure_newline_at_eof_on_save`.

You can read about why this is important in this ThoughtBot [blog post](https://robots.thoughtbot.com/no-newline-at-end-of-file).